Keas.Mvc/Startup.cs

```diff
@@ -263,6 +262,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
             app.UseRouting();
             app.UseSession();
             app.UseAuthentication();
+            app.UseMiddleware<ApiKeyMiddleware>();

```

Comment:
@sprucely For this to work correctly it had to happen after after useAuthentication?
If yes, it might be good to add that as a comment